metadata.yaml

```diff
@@ -28,7 +28,7 @@ provides:
       Provides oathkeeper deployment info to a related application
 requires:
   kratos-endpoint-info:
-    interface: kratos_endpoints
+    interface: kratos_info
```

Comment:
suggestion(non-blocking): could we just get rid of the `kratos-endpoint-info` naming altogether so we perform a clean cut with the previous naming?